test/dataframerow.jl

```diff
@@ -42,27 +47,41 @@ end
 
     r = DataFrameRow(df, 2, :)
     @test r[:] === r
```

Comment:
BTW, shouldn't this return a named tuple?

Reply:
Well - we have that `r[cols]` is a `DataFrameRow`. We could change the rules so that:
* `dfr[cols]` is a `NamedTuple`
* `@view dfr[cols]` is a `DataFrameRow`

This is a separate PR I think. Do you think we should change this? I am not really sure what is best, and in general I would leave it as is, as it is not a top priority operation anyway. Returning `NamedTuple` from `dfr[cols]` is kind of inconsistent with the fact that we allow `dfr[cols] = something`.

Reply:
Scratch that, for some reason I thought `dfr[cols]` returns a named tuple.